jobs/config.json

```diff
@@ -6726,6 +6726,26 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubernetes-e2e-gke-device-plugin-gpu": {
```

Comment:
Also need one for gce. You will need a corresponding project and quota for that as well.